cluster/gce/win1803/common.psm1

```diff
@@ -86,5 +86,69 @@ function Get-InstanceMetadataValue {
   }
 }
 
+function Validate-Hash {
```

Comment:
Add a comment that the hash is SHA1, or change the function name to indicate this.

Reply:
Done.